tests/functional/merge/cram/merge-sequences.t

```diff
@@ -0,0 +1,60 @@
+SETUP
+
+  $ source "$TESTDIR"/_setup.sh
+
+BASIC USAGE
```

Comment:
🐛 at 81b22d79. when a FASTA doesn't have a trailing new line it results in a malformed merged FASTA.

e.g. 2 fasta files each without trailing new lines:
```
# A.fasta
>A
ATGC
# B.fasta
>B
ATGC
```

```console
$ augur merge  --sequences A=A.fasta B=B.fasta  --output-sequences -
>B
ATGC>AATGC
```

I also find it strange that B comes before A in the output - is this on purpose?